apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -57,6 +59,12 @@ function StakeForm({ coinBalance, coinType, epoch }: StakeFromProps) {
         );
     }, [setFieldValue, gasBudgetInMist, isLoading]);
 
+    // Reward will be available after 2 epochs
+    const startEarningRewardsEpoch = epoch ? Number(epoch) + 2 : 0;
```

Comment:
could we pull `2` into a constant perhaps? that way the code is more self-documenting and understandable if needed else where or the value changes